src/sections/ui/tabs/Tabs.tsx

```diff
@@ -0,0 +1,15 @@
+import { PropsWithChildren } from 'react'
+import { Tab } from './Tab'
+import { Tabs as TabsBS } from 'react-bootstrap'
+
+interface TabsProps {
+  defaultActiveKey: string
+}
+
+function Tabs({ defaultActiveKey, children }: PropsWithChildren<TabsProps>) {
```

Comment:
Shouldn't this have a heading of "Props"? This is what I see:

![Screen Shot 2023-04-18 at 10 17 42 AM](https://user-images.githubusercontent.com/21006/232806028-6ead8f1a-905f-4376-b8eb-cebed1492c22.png)

Reply:
Hi @pdurbin ! The Props heading is missing because this branch is older than the #40 mdx storybook docs template. Then the template with the Props heading is missing in this branch but once this is merged it should look good in dev

Reply:
@MellyGray yes, you're right. I hope you don't mind but I did `git rebase develop` on this PR and then force pushed it. The Props heading is there now:

![Screen Shot 2023-04-18 at 2 21 51 PM](https://user-images.githubusercontent.com/21006/232868923-cdec8436-69d4-4ee8-bef5-c4ed471aa751.png)